test/testTrackPropagationPipeLine.cxx

```diff
@@ -0,0 +1,128 @@
+/// -------------------------------------------------------------------------------------+
+/// VectorFlow unit test. Generation + Propagation.                                      |
+///                                                                                      |
+/// This test is intended to show the usage of the PipelineFlow interface of VectorFlow. |
+/// The example propagates charged particles in a constant field in several steps,       |
+/// until they reach a sphere with a given radius centered in the event vertex.          |
+///                                                                                      |
+/// The pipeline flow has only one task (TaskPropagator) and the data type to be used    |
+/// are the events that are generated in the event-loop.                                 |
+/// After each execution, the pipeline is cleared and the next event is processed.       |
+/// -------------------------------------------------------------------------------------+
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <vectorFlow/PipelineFlow.h>
+#include "CocktailGenerator.h"
+#include "Event.h"
+#include "Track.h"
+#include "Particle.h"
+#include "vectorFlow/SystemOfUnits.h"
+#include "base/Vector3D.h"
+#include "HelixPropagator.h"
+#include "SimpleStepper.h"
+
+using namespace vectorflow;
+using Event_t = CocktailGenerator::Event_t;
+
+struct TaskPropagator : public Work<Event_t, std::vector<Event_t*>> {
+  // Propagator task needs a stepper
+  trackml::SimpleStepper* stepper;
```

Comment:
Should read fStepper, data members start with f and then camel case

Reply:
Fixed. Changed stepper to fStepper.